public/app/percona/add-instance/components/AddRemoteInstance/FormParts/FormParts.test.tsx

```diff
@@ -9,6 +9,7 @@ import { MainDetailsFormPart } from './MainDetails/MainDetails';
 import { ExternalServiceConnectionDetails } from './ExternalServiceConnectionDetails/ExternalServiceConnectionDetails';
 import { getMount } from 'app/percona/shared/helpers/testUtils';
 import { Databases } from 'app/percona/shared/core';
+import { dataTestId } from '@percona/platform-core';
```

Comment:
I see a bunch of disabled unit tests in this suite, it would be great to:
1/ re-enable them,
2/ migrate from enzyme, if possible

Reply:
@atymchuk, I'm going to see why Tiago turned them off and try to turn them back on. But migration is probably a separate refactoring task. I want to ask @denisok, should I return the tests disabled before me as part of this task, or will we put it on a separate task?

Reply:
@atymchuk could we do it in separate PR? we need this one to land asap

Reply:
yes, sure!

Reply:
so, @atymchuk can you please approve if the rest is ok?